src/Bangazon/Managers/MenuManager.cs

```diff
@@ -17,6 +17,7 @@ public class MenuManager {
             Console.WriteLine("*************************************************");
             Console.WriteLine("1. Create a customer account");
             Console.WriteLine("2. Choose active customer");
+            Console.WriteLine("3. Create a payment option");
```

Comment:
For option 3, I think it should say "Create a payment option for a customer". Also "Please choose a customer first" is better if user clicks number 3 rather than what we have)